src/libs/antares/study/parts/hydro/container.h

```diff
@@ -34,6 +34,98 @@
 
 namespace Antares::Data
 {
+/*!
+ ** \brief Final Reservoir Level data for a single area
+ */
+class FinalReservoirLevelRuntimeData
+{
+public:
+    // FInal reservoir level runtime data
+    std::vector<bool> includeFinalReservoirLevel;
```

Comment:
Here it's not clear what the index should be.

Please note that some structures are shared across threads (see numSpace).

Reply:
added explanation for vector indexes.
Tested for parallel work. Works fine.
The pre-checks are completed before parallelization, I checked.
The point is not to break the simulation when it is already running (parallel or not). 
The simulation is aborted before the simulation starts if something is wrong with the final reservoir data.